vllm/worker/model_runner.py

```diff
@@ -7,6 +7,9 @@
 import numpy as np
 import torch
 import torch.nn as nn
+from flashinfer import BatchDecodeWithPagedKVCacheWrapper
+from flashinfer.decode import CUDAGraphBatchDecodeWithPagedKVCacheWrapper
+from flashinfer.prefill import BatchPrefillWithPagedKVCacheWrapper
```

Comment:
This seems to make flashinfer a must-to-have dependency even flashinfer is not used.

Reply:
Will change it to try, except to remove the must-to-have dependency